cylc_singleuser.py

```diff
@@ -133,7 +133,7 @@ def main():
                         default=8888)
     parser.add_argument('-s', '--static', action="store", dest="static",
                         required=True)
-    args = parser.parse_args()
+    args, _ = parser.parse_known_args()
```

Comment:
Is `_` assigned to the unknown arguments here?

Reply:
Yes. Used it as otherwise the IDE linter complains about an unused variable. But happy to change it to a more descriptive value if you think it's better

Reply:
Can you do this instead?

```python
args = parser.parse_known_args()[0]
```

On the other note, in your original description, you mentioned the `--debug` option. I guess this is not used yet? Ever?

Reply:
>Can you do this instead?

Done! Good idea!

>On the other note, in your original description, you mentioned the --debug option. I guess this is not used yet? Ever?

Not yet. There are other settings in the `jupyterhub_config.py` like `c.JupyterHub.hub_connect_ip` (if I recall correctly) that cause the spawner to pass `--ip ?.?.?.?` as parameter.

The current `cylc_singleuser.py` code started with no settings I think. Then we added one for the port, then another for the static content. So we may use debug later when we start doing more logging in the UI Server.

Reply:
Understood.